src/Models/Tool.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace SolutionsService.Models
+{
+    public class Tool
```

Comment:
Why does tool not have a solution, but the other sub-models do?

Reply:
Because I forgot